### ISSUE TITLE
Installation fails under Node.js v7 if --engine-strict=true

### ISSUE THREAD

**Issue by k7sleeper** [tjunnone/npm-check-updates#341]
Having set `--engine-strict=true` ncu installation fails under Node.js 7 with the following message
```
>npm install npm-check-updates -g
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "npm-check-updates" "-g"
npm ERR! node v7.8.0
npm ERR! npm  v4.2.0
npm ERR! code ENOTSUP

npm ERR! notsup Unsupported engine for got@5.7.1: wanted: {"node":">=0.10.0 <7"} (current: {"node":"7.8.0","npm":"4.2.0"})
npm ERR! notsup Not compatible with your version of node/npm: got@5.7.1
npm ERR! notsup Not compatible with your version of node/npm: got@5.7.1
npm ERR! notsup Required: {"node":">=0.10.0 <7"}
npm ERR! notsup Actual:   {"npm":"4.2.0","node":"7.8.0"}

npm ERR! Please include the following file with any support request:
npm ERR!     C:\ProgramData\node.js\npm-cache\_logs\2017-04-20T09_08_12_166Z-debug.log
```

**Comment by raineorshine** [comment]
The issue appears to be from the engine limitation in `got@5.7.1`:
https://github.com/sindresorhus/got/blob/v5.7.1/package.json#L20

This is solved in a newer version of `got`, but this is a deeply nested dependency in ncu.
`update-notifier > latest-version > package-json > got`

I don't know the canonical way to handle this, although I would tentatively assert that is outside of ncu's territory. The dependencies could be contacted one-by-one to remove the upper bound. I'm open to hearing others' ideas.

**Comment by eighty4** [comment]
It looks like there are two transitive deps on got, both coming in through update-notifier, which is out of date in ncu. However, updating update-notifier to the latest version will fix one of the transitive deps to got@6.7.1, but the other is still going to be 5.7.1. It looks like updating update-notifier > xo > meow (which the latest version is got > 6) would resolve it.

**Comment by raineorshine** [comment]
Closing due to inactivity, but let me know if you believe something else should be done on npm-check-updates' side.
